### PR TITLE
IGVF-86 docs

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/document.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "enum": [
+        "FACS",
+        "immunoblot",
+        "immunofluorescence",
+        "immunoprecipitation",
+        "mass spectrometry",
+        "PCR",
+        "restriction digest",
+        "RT-qPCR",
+        "Sequencing"
```
Comment (jenjou) [inline src/igvfd/schemas/changelogs/document.md:25]: To be consistent with the other enums, "sequencing" could be lower case.

**Comment by jenjou** [comment]
There was no test_schema_documents created when we initially added the Documents schema, so perhaps we should add that in this ticket. The fixture experimental_protocol_document is also unused by any tests.

File: src/igvfd/schemas/changelogs/document.md (hunk 1 above)
Comment (gabdank) [reply src/igvfd/schemas/changelogs/document.md:25]: ok

**Review by jenjou — APPROVED** [review]
Test was added and other comments addressed.

**Pull request merged**